splice_machine/src/test/java/com/splicemachine/derby/utils/SpliceDateFunctionsTest.java

```diff
@@ -265,9 +268,41 @@ public void toChar() throws ParseException {
         String format = "MM/dd/yyyy";
         String compare = "06/24/2014";
         assertEquals(compare, SpliceDateFunctions.TO_CHAR(date, format));
+
         assertEquals(compare, SpliceDateFunctions.TIMESTAMP_TO_CHAR(timestamp, format));
     }
 
+    @Test
+    public void toCharInvalidFormat() throws ParseException, StandardException {
+        DateFormat formatter = new SimpleDateFormat("yyyy/MM/dd");
+        Date date = new Date(formatter.parse("2014/06/24").getTime());
+        Calendar calendar = Calendar.getInstance();
+        calendar.setTime(date);
+        Timestamp timestamp = new Timestamp(calendar.getTimeInMillis());
+        String format = "invalid_format";
+        String compare = "06.24.2014";
+
+        try {
+            SpliceDateFunctions.TO_CHAR(date, format);
+            Assert.fail("expected exception");
+        } catch(Exception e) {
+            Assert.assertEquals("ERROR 22018: Invalid character string format for type datetime.\n" +
```

Comment:
Can you catch the specific StandardException and assert on the SqlState instead of the whole `e.toString()`? I think that's more concise and clear.